infrastructure/zk/src/hyperchain_wizard.ts

```diff
@@ -44,34 +45,30 @@ export interface BasePromptOptions {
     skip?: ((state: object) => boolean | Promise<boolean>) | boolean;
 }
 
-// An init command that allows configuring and spinning up a new hyperchain network.
-async function initHyperchain(runObservability: boolean, deploymentMode: DeploymentMode) {
-    await announced('Initializing hyperchain creation', setupConfiguration(runObservability));
-
-    const deployerPrivateKey = process.env.DEPLOYER_PRIVATE_KEY;
-    const governorPrivateKey = process.env.GOVERNOR_PRIVATE_KEY;
-    const deployL2Weth = Boolean(process.env.DEPLOY_L2_WETH || false);
-    const deployTestTokens = Boolean(process.env.DEPLOY_TEST_TOKENS || false);
-    const governorAdrress = ethers.utils.computeAddress(governorPrivateKey!);
-    const initArgs: InitArgs = {
-        skipSubmodulesCheckout: false,
-        skipEnvSetup: true,
-        runObservability: runObservability,
-        governorPrivateKeyArgs: ['--private-key', governorPrivateKey],
-        deployerL2ContractInput: {
-            args: ['--private-key', deployerPrivateKey],
-            includePaymaster: false,
-            includeL2WETH: deployL2Weth
-        },
-        testTokens: {
-            deploy: deployTestTokens,
-            args: ['--private-key', deployerPrivateKey, '--envFile', process.env.CHAIN_ETH_NETWORK!]
-        },
-        deployerPrivateKeyArgs: ['--private-key', deployerPrivateKey, '--owner-address', governorAdrress],
-        deploymentMode
-    };
+// PLA:681
+let isLocalhost = false;
 
-    await init(initArgs);
+// An init command that allows configuring and spinning up a new hyperchain network.
+async function initHyperchain(envName: string, runObservability: boolean, validiumMode: boolean) {
+    await announced('Initializing hyperchain creation', setupConfiguration(envName, runObservability));
+    await init.initHyperCmdAction({ skipSetupCompletely: false, bumpChainId: true, runObservability, validiumMode });
+
+    // if we used matterlabs/geth network, we need custom ENV file for hyperchain compose parts
+    // This breaks `zk status prover` command, but neccessary for working in isolated docker-network
+    // TODO: Think about better implementation
+    // PLA:681
+    if (isLocalhost) {
+        env.modify(
+            'ETH_CLIENT_WEB3_URL',
+            'http://geth:8545',
```

Comment:
we moved to 'reth' a while ago - does this initHyperchain code still work?

Reply:
cc @kelemeno

Reply:
Fixed here https://github.com/matter-labs/zksync-era/pull/1766